context/app/static/js/components/Search/DownloadButton/style.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import Button from '@material-ui/core/Button';
+
+const StyledButton = styled(Button)`
+  margin-left: ${(props) => props.theme.spacing(1)}px;
+  background-color: white;
+  height: 40px;
+  margin-right: ${(props) => props.theme.spacing(1)}px;
+  border-radius: 3px;
+  border: 1px solid ${(props) => props.theme.palette.secondary.main};
```

Comment:
Looking at the design, I don't think there is a background color. So I think you can get most of this passing the `variant="outlined" color="primary"` props to the button component.

Reply:
Also, I wouldn't add a fixed height for the button. I would add padding if you want to increase the size incase anyone increases the default font size in their browser.

Reply:
Thanks: I've simplified the CSS to this:
```
  margin: 0 ${(props) => props.theme.spacing(1)}px;
  height: 40px;
``` 
which gives this:
<img width="286" alt="Screen Shot 2021-09-13 at 9 54 11 AM" src="https://user-images.githubusercontent.com/730388/133096522-89f7a4b0-0024-4641-a6ea-bff2b309a2b9.png">

but if I remove the height, it's too short. `TilesSortDropdown` also has `height: 40px`... Does it make sense for them to work the same way?

Reply:
Ah ok, keep it how it is and hopefully sometime I'll take a look.